store/store.go

```diff
@@ -233,6 +240,125 @@ func (s *SnapUbuntuStoreRepository) checkStoreResponse(resp *http.Response) {
 	}
 }
 
+/*
+purchase encapsulates the purchase data sent to us from the software center agent.
+
+This object type can be received in response to requests about the user's current
+purchases, and also when making purchase requests.
```

Comment:
This sentence can be removed. The first sentence points the same thing in a more compact way, and will remain correct for longer.

Also, can we please use the // style of comments here? It's okay that it's a bit longer.